samples/Samples.SqlServer/Program.cs

```diff
@@ -6,6 +6,7 @@ namespace Samples.SqlServer
 {
     class Program
     {
+        [LoaderOptimization(LoaderOptimization.MultiDomainHost)]
```

Comment:
Before implementation changes (7094adc9e3ff353fce4fe41417af483836a7fcdd):
Using `LoaderOptimization.MultiDomainHost` loads `System.Data` into the shared domain (instead of the default domain), causing us to not profile the calls to `System.Data.Common.DbCommand.ExecuteReader()` before these PR changes.

<img width="1147" alt="windows-integration-test-failure-sqlserver" src="https://user-images.githubusercontent.com/13769665/65171327-6f609800-d9ff-11e9-99a3-cfe914e492e3.PNG">

Reply:
We are still unable to profile these calls unless the profiler is loaded into the GAC.

Reply:
For this reason, I'm proposing that we change the MSI back to installing our net45 components into the GAC.